tactics/really_bad_selfatari.c

```diff
@@ -0,0 +1,402 @@
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#define QUICK_BOARD_CODE
+
+#define DEBUG
+#include "board.h"
+#include "debug.h"
+#include "mq.h"
+#include "random.h"
+#include "tactics/1lib.h"
+#include "tactics/nakade.h"
+#include "tactics/really_bad_selfatari.h"
+
+/* is_bad_selfatari() clone that only cares about not losing big groups.
+ * Not terribly fond of having all this code duplicated. Oh well ... */
```

Comment:
Well, I'm going to resist a bit here. :) I'd like to really explore why do we need this code duplication at all, since I'm at loss. I can see only very trivial differences from the original selfatari code, obscured just by some refactoring which was done just in this copy of the code, and that'll further make maintenance more difficult. Can't we just pass an extra flag to is_bad_selfatari()? What is a "big group", 3+ stones?